release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1113,7 +1150,12 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-samples-bug-fixes"]
 ==== Samples
 
-* Before this update, if the Cluster Samples Operator encountered an `APIServerConflictError` error, it reported `sample-operator` as having `Degraded` status until it recovered. Momentary errors of this type are not unusual during upgrades and were causing undue concern for administrators monitoring the Operator status. With this update, if the Operator encounters a momentary error, it no longer indicates `openshift-samples` as having `Degraded` status and retries to connect to the API server. Momentary shifts to `Degraded` status should no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1993840[*BZ#1993840*])
+* Before this update, if the Cluster Samples Operator encountered an `APIServerConflictError` error, it reported `sample-operator` as having `Degraded status` until it recovered. Momentary errors of this type are not unusual during upgrades and were causing undue concern for administrators monitoring the Operator status. With this update, if the Operator encounters a momentary error, it no longer indicates `openshift-samples` as having `Degraded status` and retries to connect to the API server. Momentary shifts to `Degraded status` should no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1993840[BZ#1993840])
+
+* Before this update, various allowed and block registry configuration options of the cluster image configuration might block the Cluster Samples Operator from creating image streams. When that happened, the samples operator marked itself as degraded, which impacted the general {product-title} install and upgrade status.
++
+The Cluster Samples Operator can bootstrap itself as removed in various circumstances. With this update, these circumstances include when the image controller configuration parameters prevent the creation of image streams by using the default image registry or the image registry specified by the `samplesRegistry` setting. The Operator status also indicates that the cluster image configuration is preventing the creation of the sample image streams. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002368[BZ#2002368])
```

Comment:
```suggestion
The Cluster Samples Operator can bootstrap itself as removed in various circumstances. With this update, these circumstances include when the image controller configuration parameters prevent the creation of image streams by using either the default image registry or the image registry specified by the `samplesRegistry` setting. The Operator status also indicates that the cluster image configuration is preventing the creation of the sample image streams. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002368[BZ#2002368])
```
- I don't understand what "bootstrap itself as removed" means. Is this correct terminology for this feature?
- Added "either" to emphasize that there are two alternative options

Reply:
Updated. Thanks.